examples/src/main/java/io/dapr/examples/secrets/README.md

```diff
@@ -0,0 +1,112 @@
+# Dapr's Secret Store Sample
+
+In this sample, we'll see how to retrieve a secret using Dapr's Java SDK. 
+This sample includes two files:
+
+* docker-compose-vault.yml (Starts Hashicorp's Vault as a container)
+* SecretClient.java (Reads a secret from Dapr's Secret Store)
+* Existing Dapr component file in `< repo dir >/examples/components/hashicorp_vault.yaml`
+* Existing token file in `< repo dir >/examples/.hashicorp_vault_token` (Consumed by `daprd`'s vault component above)
+
+Visit [this](https://github.com/dapr/docs/tree/master/concepts/secrets) link for more information about secret stores in Dapr.
+ 
+## Binding sample using the Java-SDK
```

Comment:
It looks like this should be about secret stores, not bindings.

Reply:
Yes, copy and pasta.

Reply:
Done.